jobs/resend_invitation_email/scheduler.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+package resend_invitation_email
+
+import (
+	"time"
+
+	"github.com/mattermost/mattermost-server/v5/app"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+const ResendInvitationEmailJob = "ResendInvitationEmailJob"
+
+type ResendInvitationEmailScheduler struct {
+	App *app.App
+}
+
+func (rse *ResendInvitationEmailJobInterfaceImpl) MakeScheduler() model.Scheduler {
+	return &ResendInvitationEmailScheduler{rse.App}
+}
+
+func (s *ResendInvitationEmailScheduler) Name() string {
+	return ResendInvitationEmailJob + "Scheduler"
+}
+
+func (s *ResendInvitationEmailScheduler) JobType() string {
+	return model.JOB_TYPE_RESEND_INVITATION_EMAIL
+}
+
+func (s *ResendInvitationEmailScheduler) Enabled(cfg *model.Config) bool {
+	// Must be a cloud installation
+	return s.App.Srv().License() != nil && *s.App.Srv().License().Features.Cloud
```

Comment:
We do not have to limit this functionality to cloud only

Reply:
Oh yes. I had forgotten about this review from enterprise repo